default.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <remote name="origin" fetch="https://git.launchpad.net/~snappy-hwe-team/snappy-hwe-snaps/+git" />
+  <default revision="master" remote="origin"  sync-j="4" />
```

Comment:
I don't agree. Master always reflects our development branch. Once a snap is pushed to beta/candidate everything is tagged and merged into the stable branch. With that we can always ensure we have a well-known state of our documentation for our snaps being in beta/candidate/stable. So if we select a default here for the time being it should be 'stable' and not master. Otherwise we risk that not yet available features become available on docs.ubuntu.com and that is nothing we want, even not with this first initial version.